include/grpcpp/ext/csm_observability.h

```diff
@@ -97,7 +95,15 @@ class CsmObservabilityBuilder {
   std::unique_ptr<grpc::internal::OpenTelemetryPluginBuilderImpl> builder_;
 };
 
+namespace experimental {
+// TODO(yashykt): Remove this once no longer needed.
```

Comment:
Who is using these right now?

Reply:
just in case someone is using it. They'll get warnings first, and then in coming releases we can remove it